src/appmixer/ai/openai/TransformTextToJSON/component.json

```
@@ -0,0 +1,65 @@
+{
+    "name": "appmixer.ai.openai.TransformTextToJSON",
+    "author": "Appmixer <info@appmixer.com>",
+    "description": "Extract structured JSON data from text using AI.",
+    "auth": {
+        "service": "appmixer:ai:openai"
+    },
+    "inPorts": [{
+        "name": "in",
+        "schema": {
+            "type": "object",
+            "properties": {
+                "text": { "type": "string" },
+                "jsonSchema": { "type": "string" },
+                "model": { "type": "string" }
+            },
+            "required": ["text", "jsonSchema"]
+        },
+        "inspector": {
+            "inputs": {
+                "text": {
+                    "label": "Text",
+                    "type": "textarea",
+                    "index": 1,
+                    "tooltip": "The text from which to extract structured JSON data. Example: <code>John is 25 years old.</code>."
+                },
+                "jsonSchema": {
+                    "label": "Output JSON Schema",
+                    "type": "textarea",
+                    "index": 2,
+                    "tooltip": "The schema that defines the structure of the output JSON. Use <a target=\"blank\" href=\"https://json-schema.org/\">JSON Schema format</a>. Example: <code>{\"type\":\"object\",\"properties\":{\"name\":{\"type\":\"string\"}, \"age\":{\"type\":\"number\"}}}</code>. It must be a valid JSON schema and must be of \"type\": \"object\". If you want to produce an array, you can nest the array under an object property of type array. Example: <code>{\"type\":\"object\",\"properties\":{\"contacts\":{\"type\":\"array\",\"items\":{\"type\":\"object\",\"properties\":{\"name\":{\"type\":\"string\"},\"age\":{\"type\":\"number\"}}}}}}</code>."
+                },
+                "model": {
+                    "type": "text",
+                    "index": 3,
+                    "label": "Model",
+                    "tooltip": "ID of the model to use.",
+                    "defaultValue": "gpt-4o-2024-08-06",
```

Same as above. Or is there a reason for this specific version?

```suggestion
                    "defaultValue": "gpt-4o",
```